static/css/timetravel.css

```diff
@@ -1,25 +1,64 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body,
 html {
   width: 100%;
   height: 100%;
   margin: 0;
   padding: 0;
+  color: #F2F2F2;
 }
 
-.row-container {
+a svg {
+ fill: #48d597;
+}
+
+hr {
+  border: 0;
+  height: 0;
+  border-top: 1px solid rgba(0, 0, 0, 0.1);
+  border-bottom: 1px solid rgba(255, 255, 255, 0.2);
+  margin-bottom: 1rem;
+}
+
+/* utils  */
+
+.-text--fade {
+  opacity: 0.4;
+}
+
+.container {
+  position: relative;
   display: flex;
-  width: 100%;
+  flex-flow: column nowrap;
   height: 100%;
-  flex-direction: column;
-  background-color: #222200;
-  overflow: hidden;
 }
 
-.second-row {
-  flex-grow: 1;
+.display-wrapper {
+  flex: 5;
   border: none;
-  margin: 0;
-  padding: 0;
+}
+
+.history {
+  position: relative;
+  flex: 0 1 auto;
+  padding: 2rem;
+  background: rgba(18, 18, 18, 1);
+  color: #bbb;
+  border-left: 1px solid rgba(255, 255, 255, 0.12);
+  overflow: hidden;
+}
+
+.history__controller {
+  position: relative;
+  margin-bottom: 0.5rem;
+  padding: 1rem 0;
+  display: flex;
+  justify-content: space-between;
 }
 
 .history__controller > button {
```

Comment:
I'd add `font-size: 0.8rem` to ensure that all browsers have the same size (My Chrome layout has buttons with 13px by default, instead of 11px on Firefox or Safari), which should equal to ~12px with the current html font-size.

Just a tad bigger, but more readable.

Reply:
Good catch...